api/envoy/type/tracing.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto3";
+
+package envoy.type;
+
+option java_outer_classname = "TracingProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.type";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Tracing]
+
+// Describes custom tags for the active span.
+message TracingCustomTag {
+  // Literal type custom tag with literal value for tag value.
+  message Literal {
+    // literal value to populate the tag value.
+    string value = 1 [(validate.rules).string = {min_bytes: 1}];
+  }
+
+  // Environment type custom tag with environment name and default value.
+  message Environment {
+    // environment variable name to obtain the value to populate the tag value.
+    string name = 1 [(validate.rules).string = {min_bytes: 1}];
+
+    // used to populate the tag value if the environment variable is not found.
```

Comment:
Should this be required? If not, what is the behavior if this is empty? Does the tag get populated with an empty value? Not populated? Please document.

Reply:
commented